src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        Random random = new Random();
+        int randomInt = random.nextInt(100);
+        ball.setNumber(randomInt);
+        ball.setColor(ColorSupplier.getRandomColor());
+        return ball;
+    }
```

Comment:
as mentioned in common mistakes checklist, we should avoid static methods - remove static modifier from this method and from getRandomColor, make 100 a class constant, ColorSupplier and Random a class fields - this way we won't recreate them needlessly per each method invocation